src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -264,5 +263,11 @@ export default (opts) => {
     trendingArtistsLoader: gravityLoader("artists/trending"),
     userByEmailLoader: gravityLoader("user", {}, { method: "GET" }),
     userByIDLoader: gravityLoader((id) => `user/${id}`, {}, { method: "GET" }),
+    quizLoader: gravityUncachedLoader(`user_art_quiz`, {}, { method: "GET" }),
```

Comment:
An alternative invocation of this line which would obviate the need for the other modifications to the factory, could be:

```
quizLoader: gravityUncachedLoader(`user_art_quiz`, {
      xapp_token: opts.appToken,
    })
```

This works by propagating the incoming token via a query param (called `xapp_token`) to Gravity. This isn't _great_ b/c query params aren't really obfuscated at all and frequently get logged (and this would be an app token with some privileges associated with a trusted email client).

So, I do like the factory changes! Bold 😆 , but I wanted to propose the much simpler one-line alternative in case it appealed to you or you wanted to iterate.

As a note of caution (I will try to do this as well), it might be nice to make sure there are no runtime issues with existing invocations of the uncached loader (looks like it's used in a few places). What I mean is to basically exercise those loaders with these changes (for instance, query for something that uses `savedArtworksLoader`, etc.)